src/components/Data/Documents/Page.vue

```diff
@@ -486,6 +486,7 @@ export default {
         this.deleteModalIsLoading = false
       } catch (e) {
         this.$store.commit(SET_TOAST, { text: e.message })
+        console.error(e)
```

Comment:
Unwanted `console.error` ?

Reply:
Wanted.
We need to see the errors in order to debug them. It is therefore completely useless to display the message in the toast without having the trace in the console...

Reply:
We never show any trace in the admin console and I think if you want to show traces we should do that only in DEV mode and not show traces in production mode.

Reply:
True. Even if completely overkill, since we use this tool to manage one's own instances.
An still, the message in the toast is completely.

But I remember I found a Vuejs logger plugin that enabled to have logs being displayed under given conditions (like you said, dev mode or test - based on env vars).

Reply:
That would be nice! I found this one which look promising https://www.npmjs.com/package/vuejs-logger

Reply:
:heavy_check_mark: Done!